src/daemon/daemon.cpp

```diff
@@ -1990,11 +1990,11 @@ try // clang-format on
             continue;
         }
 
-        auto target_path = path_entry.target_path();
+        auto target_path = QDir::cleanPath(QString::fromStdString(path_entry.target_path()));
```

Comment:
Hey @sharder996, why don't keep `target_path` as a `std::string`? This way, the rest of the code stays unchanged.

Reply:
Overall that would result in less conversion. Not sure which is more icky, `fromStdString().toStdString()` or multiple `toStdString()` lol

Reply:
Yeah, it's not nice to have two string types and convert back and forth between them. But I like how it is now, because the diff is only two lines.